.github/workflows/chaos.yml

```diff
@@ -23,19 +23,22 @@ jobs:
       - uses: actions/checkout@v2
       - run: echo $HOME
       - run: echo $PATH
-      - run: |
+      - name: Set Npm Config
+        run: |
           npm config set -- '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
```

Comment:
Let's remove all mention of the `NPM_TOKEN`. We shouldn't need it anymore as `gauntlet-core` is now public. We still need to `yarn install` though :)